lib/game.rb

```diff
@@ -99,6 +106,29 @@ def comp_turn
         break
       end
     end
+    puts "Computer shot #{coordinate}"
+    coordinate
+  end
+
+  def player_valid_turn?(coordinate)
+    if @player_2.board.valid_coordinate?(coordinate)
+      @player_1.board.cells[coordinate].hit == false
+    else
+      puts "You can't do that, can't you see it's not an option?????"
```

Comment:
Ya maybe not the nicest thing to say as a message. LOL Maybe we can reword that a little. Haha